source/installing.rst

```diff
@@ -36,45 +47,13 @@ For easiest installation, if you have one of the listed supported distributions,
    .. code-block:: bash
 
       curl -s https://packagecloud.io/install/repositories/pufferpanel/pufferpanel/script.deb.sh | sudo bash
-      sudo apt-get install pufferpanel
-      sudo systemctl enable pufferpanel         
```

Comment:
Unless packagecloud is doing something new, the service needs to be enabled and started still

Reply:
That was redundant as its also covered by the starting the panel section.